### PR TITLE
Feature#33 about us page

### PR DESCRIPTION
**Pull request by AangelF**
# Description

What did you implemented/Why did you implemented this:

Create a about us page to explain the user about our page web and the services can provide it.

## Screenshots

image

### CODE REVIEW

**Review by DiegoMoralesBravo — APPROVED** [review]

File: app/helpers/about_us_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module AboutUsHelper
+end
```
Comment (JuanequeX) [inline app/helpers/about_us_helper.rb:2]: if this helper is unnecessary please remove it

File: app/views/about_us/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <a href="#">
+          <%= image_tag "xiaomi-logo.png" , alt: "Xiaomi" , class: "p-3" %>
+        </a>
+        <a href="#">
+          <%= image_tag "motorola-logo.png" , alt: "Motorola" , class: "p-3" %>
+        </a>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (JuanequeX) [inline app/views/about_us/index.html.erb:33]: add a blank line at the end of this file

File: app/views/partials/_footer.html.erb [hunk 3]
```diff
@@ -34,7 +34,7 @@
         </div> 
 
         <div class="col-md-4 col-lg-3 col-xl-3 mx-auto mb-md-0 mb-4">
-          <a href="#"><h5 class="fw-bold"> Sobre Nosotros </h5></a>
+          <a href="#"><h5 class="fw-bold text-black"> Sobre Nosotros </h5></a>
```
Comment (JuanequeX) [inline app/views/partials/_footer.html.erb:37]: remove the `h5` label is unnecessary if you have an `a` wrapped in it

File: test/controllers/about_us_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AboutUsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (JuanequeX) [inline test/controllers/about_us_controller_test.rb:7]: it will be cool if adding tests for this controller

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by oscaregr — APPROVED** [review]

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**